src/hotspot/share/runtime/arguments.cpp

```diff
@@ -1297,15 +1314,22 @@ bool Arguments::process_argument(const char* arg,
       warning("Ignoring option %s; support was removed in %s", stripped_argname, version);
       return true;
     }
+    const AliasedObsoleteLoggingFlag* obs_replacement;
+    if (has_plus_minus && (obs_replacement = removed_product_logging_flag_name(stripped_argname)) != NULL) {
+      warning("Ignoring option %s; support was removed in %s.  Please use %s%s instead.",
+              stripped_argname,
+              obs_replacement->version,
+              obs_replacement->log_name,
+              *arg == '+' ? obs_replacement->tag_name : "off");
+      return true;
+    }
```

Comment:
I see you left this empty in case we change more flags to logging flags.  I don't see any Trace flags left that users would care about that merit this amount of helpfulness.  I think this should be removed too.